### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by enaut** [stalwartlabs/website#4]
Hello,

I'm currently trying to set up stalwart. The documentation is basically very good at explaining the options thank you for that! However, I'm missing which file I would add the config to. For example: https://stalw.art/docs/directory/types/sql#connection-string. Is this for the mail or the smtp-backend?

**Comment by mdecimus** [comment]
Hi,

Settings can be added to any of the toml files since they are all merged at startup. However, if you would like to add or modify a directory, this should be done under `etc/directory` directory.

**Comment by enaut** [comment]
Thank you for your kind reply. I understood the configs way better now however I still think it would be helpful to have the default paths where the setting is placed somewhere on top of the docs or the code-display - most (all?) of the settings are already somewhere meaning if I add them in the wrong place they might get overwritten by the default. By having the "opinionated best place" in docs some of these errors might vanish.

Right now I just use grep on the config directory, which works.

**Comment by enaut** [comment]
I made a rough mockup of what I mean: 
![grafik](https://github.com/stalwartlabs/website/assets/290005/32dde949-1f94-41ea-9554-784cfd48b392)

**Comment by mdecimus** [comment]
I agree, I'll add those details once the next version is released. Previously all settings were in a single file so the documentation needs to be updated.

**Comment by mdecimus** [comment]
Thanks for the feedback. v0.16 stores almost all configuration in the database (managed through the WebUI or CLI); the only file involved at startup is a small `config.json` that points at the data store. The v0.15 ambiguity about which TOML file to edit no longer applies. Closing as obsolete.
